examples/lua_REPL/repl.lua

```diff
@@ -0,0 +1,91 @@
+--[[
```

Comment:
As you mentioned it is probably not necessary to be thorough with the [Lua style guide](http://lua-users.org/wiki/LuaStyleGuide). One thing I did notice, though, is that indents are generally two spaces. This would also reduce the array size a bit.

Reply:
Another note for the future: write a "partial minifier" that preserves line numbers and variable names.

Reply:
Two-space indents make me want to rip off my eyes.

Reply:
ok. That's a fair reaction tbh